dev-docs/analytics/adWMG.md

```diff
@@ -3,9 +3,32 @@ layout: analytics
 title: adWMG
 description: adWMG Analytics Adapter
 modulecode: adWMG
+gdpr_supported: true
+usp_supported: false
+coppa_supported: false
+prebid_member: true
```

Comment:
Thanks @nyakove - I don't see AdWMG as a [prebid.org member company](https://prebid.org/membership/member-directory/#community). Please set that value to false.

Reply:
@bretg oh, changed